modules/howtos/examples/Analytics.java

```diff
@@ -122,50 +155,76 @@ public static void main(String... args) {
     {
       // tag::rowsasobject[]
       AnalyticsResult result = cluster.analyticsQuery(
-        "select * from `travel-sample` limit 10"
+        "select * from airports limit 3"
       );
       for (JsonObject row : result.rowsAsObject()) {
         System.out.println("Found row: " + row);
       }
       // end::rowsasobject[]
+      System.out.println();
     }
 
     {
-      // tag::simplereactive[]
-      Mono<ReactiveAnalyticsResult> result = cluster
-        .reactive()
-        .analyticsQuery("select 1=1");
+      // tag::handle-collection[]
+      AnalyticsResult result = cluster.analyticsQuery(
+        "SELECT airportname, country FROM `airports-collection` WHERE country='France' LIMIT 3");
+      // end::handle-collection[]
+      show("handle-collection", result);
+    }
 
-      result
-        .flatMapMany(ReactiveAnalyticsResult::rowsAsObject)
-        .subscribe(row -> System.out.println("Found row: " + row));
-      // end::simplereactive[]
+    {
+      // tag::handle-scope[]
+      Bucket bucket = cluster.bucket("travel-sample");
+      Scope scope = bucket.scope("inventory");
+      AnalyticsResult result = scope.analyticsQuery(
+        "SELECT airportname, country FROM `airports-collection` WHERE country='France' LIMIT 4");
```

Comment:
actually I'm not sure you need to do it like this way with 7.0 .. I think you can query collections directly with analytics now? (so no need to create a specific dataset for it)

Reply:
@daschl I couldn't get that to work either via the SDK or the workbench Analytics query -

e.g.  I get an error like “Cannot find dataset airline in dataverse travel-sample.inventory nor an alias with name airline!” if I try to run `select * from travel-sample.inventory.airline` (e.g. against a collection directly, instead of an analytics dataset)

So I'm very Confused! 😕 

Do you have a working simpler example, or other advice?